project/Build.scala

```diff
@@ -38,7 +38,20 @@ import org.scalajs.core.tools.linker.backend.OutputMode
 
 import sbtassembly.AssemblyPlugin.autoImport._
 
-object Build extends sbt.Build {
+/*
+ * In sbt 0.13 the Build trait would expose all vals to the shell, where you can use them in "set a := b"
+ * like expressions. This re-exposes them.
```

Comment:
Try to keep line length under 80 characters, as is the soft rule in our coding style. By the way, we start our multi-line comments on the first line, so it should look like this:
```scala
/* In sbt 0.13. the Build trait ...
 * ... This re-exposes them.
 */
```
(yes, we are coding style nazis ^^ see https://github.com/scala-js/scala-js/blob/master/CODINGSTYLE.md)